src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/PlainText.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using Microsoft.PowerFx.Core.Functions;
+using Microsoft.PowerFx.Core.Localization;
+using Microsoft.PowerFx.Core.Types;
+using Microsoft.PowerFx.Core.Utils;
+
+namespace Microsoft.PowerFx.Core.Texl.Builtins
+{
+    internal class PlainTextFunction : BuiltinFunction
+    {
+        public override bool IsSelfContained => true;
+
+        public PlainTextFunction()
+            : base("PlainText", TexlStrings.AboutPlainText, FunctionCategories.Text, DType.String, 0, 1, 1, DType.String)
+        {
+        }
+
+        public override IEnumerable<TexlStrings.StringGetter[]> GetSignatures()
+        {
+            yield return new[] { TexlStrings.PlainTextArg1 };
+        }
+    }
```

Comment:
Can we move already existing Implementation from PA?
(Luc recently moved regex function from PA so may have pointers on how to do it w/o breaking PA)
[https://msazure.visualstudio.com/OneAgile/_git/PowerApps-Client?path=/src/Cloud/DocumentServer.Core/Document/Texl/Builtins/PlainText.cs&version=GBmaster](https://msazure.visualstudio.com/OneAgile/_git/PowerApps-Client?path=/src/Cloud/DocumentServer.Core/Document/Texl/Builtins/PlainText.cs&version=GBmaster)

Reply:
not sure what this mean but this function is trying to implement the same way with PA.

Reply:
Ideally, we want PA and FX to share a TexlFunction file.
Currently this file already exists in PA repo, so Ideally we should remove it from PA repo and move it to here FX repo...

Reply:
it is the same content file. what do you want to move from PA to Fx?

Reply:
will remove this from PA after add to Fx.

Reply:
You can use the following PR as a guide:
[https://msazure.visualstudio.com/OneAgile/_git/PowerApps-Client/pullrequest/7757449?_a=files](https://msazure.visualstudio.com/OneAgile/_git/PowerApps-Client/pullrequest/7757449?_a=files)

Reply:
Thanks, will use it.